bot/exts/moderation/stream.py

```diff
@@ -0,0 +1,135 @@
+import time
+
+import discord
+from async_rediscache import RedisCache
+from discord.ext import commands, tasks
+
+from bot.bot import Bot
+from bot.constants import Guild, Roles, STAFF_ROLES, TIME_FORMATS
+
+# Constant error messages
+NO_USER_SPECIFIED = "Please specify a user"
+TIME_FORMAT_NOT_VALID = "Please specify a valid time format ex. 10h or 1day"
+TIME_LESS_EQ_0 = "Duration can not be a 0 or lower"
+USER_ALREADY_ALLOWED_TO_STREAM = "This user can already stream"
+USER_ALREADY_NOT_ALLOWED_TO_STREAM = "This user already can't stream"
```

Comment:
To make it more clear that these are errors, prefix them with `bot.constants.Emoji.cross_mark`. This is a common practice in the bot. Similarly, success messages should be prefixed with `bot.constants.Emoji.check_mark`.